src/scripts/downloadUiAssets.js

```diff
@@ -54,6 +54,28 @@ async function getVersions(baseUrl) {
   return (null);
 }
 
+async function getVersionsForAzure(wweUrl, authUrl) {
```

Comment:
Should be renamed by getVersionsForSeparatedUIAndServicePlatform

Reply:
Fixed